test/pytest/test_browser.py

```diff
@@ -22,6 +24,14 @@
     'base1/test-websocket.html',
 }
 
+MOCK_WEBSERVER_XFAIL = {
+    'base1/test-dbus.html',
+    'base1/test-external.html',
+    'base1/test-http.html',
+    'base1/test-stream.html',
+    'shell/machines/test-machines.html',
```

Comment:
So test-machines has `cockpit.dbus(null, { bus: 'internal' })` I guess that's the system bus?

Reply:
No.  That's our internal fake D-Bus inside of the bridge.  We have some internal endpoints like configuration, machines, etc. that we publish via D-Bus for convenience, rather than making them their own channel types.